ERCS/erc-7893.md

```diff
@@ -0,0 +1,706 @@
+---
+eip: 7893
+title: DeFi Protocol Solvency Proof Mechanism
+description: An interface for verifying and reporting DeFi protocol solvency status through smart contracts
```

Comment:
> An interface [...] through smart contracts

This is arguably fluff. Nearly all ERCs propose some kind of contract interface.